client/components/Overview/serverRequests.js

```diff
@@ -1,12 +1,12 @@
 import API_KEY from '../../../token.js';
```

Comment:
Shouldnt the code from this file be in API.js and therefore this can be deleted?